examples/resources/genesyscloud_telephony_providers_edges_phone/resource.tf

```diff
@@ -0,0 +1,20 @@
+resource "genesyscloud_telephony_providers_edges_phone" "test_phone" {
+  name                   = "test phone"
+  state                  = "active"
+  site_id                = "48382f1b-fbbe-4232-8dd6-42a4fa70c1b6"
```

Comment:
Randomly generated GUIDs. Not sure if I can reference another resource's ID here

Reply:
That's fine for now. Eventually I would expect sites to be available in the provider, at which point this will be able to reference them.

Reply:
> This is generic at the moment. I can change it to be specific to WebRTC if that seems better @carnellj

@ronanwatkins Keep it generic for now.  We should do a blog post after the provider is released showing how to setup a WebRTC phone